btr-web/btr-main-app/components/individual-person/SummaryTable.vue

```diff
@@ -289,10 +386,22 @@ function updateSignificantIndividual (index: number | undefined, updatedSI: SiSc
   }
   closeEditingMode()
 }
+
+/** Capitalize the first letter in the given fullname. */
+function capFirstLetterInName (fullName: string) {
```

Comment:
I think I had to do this few times too, it might be good to put it into utils, or even into the composables. Not necessary though :)